Document live, templateContent, unsafeSVG.

arthurevans opened the pull request:
Fixes #1063, #1067, #1068.

justinfagnani commented on docs/_guide/06-template-reference.md:
```diff
@@ -420,6 +424,38 @@ const template = html`
+
+In these cases if the DOM value changes, but the value set through lit-html
+bindings hasn't, lit-html won't know to update the DOM value and will leave
+it alone. If this is not what you want—if you want to overwrite the DOM
```
Add spaces here: "want-if", and should it be balanced with another "-" at the end of the clause?

arthurevans replied:
Our house style is to use em dashes with no spaces. These do look weird in monospace fonts because the em dash is the same size as a hyphen, but it looks good when it's set in a normal font.

https://developers.google.com/style/dashes
Example in the current docs: https://lit-html.polymer-project.org/guide/template-reference#template-structure

arthurevans replied:
Added the closing dash, updated the API docs to match.

justinfagnani commented on docs/_guide/06-template-reference.md:
```diff
@@ -481,6 +517,33 @@ be combined with static values:
+
+  ${templateContent(templateEl)}`;
+
+render(template, document.body);
```
Maybe leave out the render call?

arthurevans replied:
SGTM

arthurevans commented:
Should be ready to merge @justinfagnani ... Addressed comments & made minor tweaks to the API docs to match (we don't actually render the API docs for directives ATM, so this is kind of a no-op).

justinfagnani commented:
Thanks @arthurevans !
